#! /bin/sh /usr/share/dpatch/dpatch-run
## nomarch.dpatch by Kilian Krause <kilian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't optimize for i686 anywhere
## -- upstream will move to configure with 1.4

@DPATCH@
Index: asterisk-1.2.23~dfsg/Makefile
===================================================================
--- asterisk-1.2.23~dfsg.orig/Makefile	2007-08-06 12:01:58.000000000 +0200
+++ asterisk-1.2.23~dfsg/Makefile	2007-08-06 12:02:04.000000000 +0200
@@ -237,10 +237,6 @@
   ASTCFLAGS+=-I$(CROSS_COMPILE_TARGET)/usr/local/include -L$(CROSS_COMPILE_TARGET)/usr/local/lib
 endif
 
-ifneq ($(PROC),ultrasparc)
-  ASTCFLAGS+=$(shell if $(CC) -march=$(PROC) -S -o /dev/null -xc /dev/null >/dev/null 2>&1; then echo "-march=$(PROC)"; fi)
-endif
-
 ifeq ($(PROC),ppc)
   ASTCFLAGS+=-fsigned-char
 endif
